hls-plugin-api/src/Ide/Types.hs

```diff
@@ -205,6 +205,25 @@ instance PluginMethod TextDocumentCodeAction where
         , Just caKind <- ca ^. kind = any (\k -> k `codeActionKindSubsumes` caKind) allowed
         | otherwise = False
 
+instance PluginMethod TextDocumentDefinition where
+  pluginEnabled _ _ _ = True
+  combineResponses _ _ _ _ (x :| _) = x
```

Comment:
@michaelpj Don't you want to discuss the `combineResponses` function here?

Reply:
I commented on the other PR: I think this could be made more sensible, but I'm happy to just have this for now. Or do you want to sort it out properly? I think it's not 100% trivial because of the weird type the spec gives it (`Location[] | LocationLink[] | null`), but in principle I think you could merge things (probably by demoting `LocationLink`s into `Location`s if you have to).

Reply:
i.e. we could do it in a follow up PR so we unblock your other one.

Reply:
Yeah, that sounds like it aligns better with my goals :sweat_smile: